src/shared/components/annotation/oncokb/AbstractItem.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import styles from '../styles/oncokb/listGroupItem.module.scss';
+import {ArticleAbstract} from "../../../api/generated/OncoKbAPI";
+
+export default class AbstractItem extends React.Component<ArticleAbstract> {
```

Comment:
I think AbstractItem is, to put it mildly, too vague.  We shouldn't have to look at folder path to gain specifity.

Reply:
Would be clarify the purpose of the component after renaming to `ArticleAbstractItem`? I think the only confusing part is the component using a type from OncoKB?

Reply:
Ah, I see, it's "astract" in a sense of a "scientific abstract".    Yeah, I think ArticleAbstractItem would be better.